ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Dars.scala

```diff
@@ -3,14 +3,14 @@
 
 package com.daml.ledger.api.testtool.infrastructure
 
+import com.daml.ledger.test.TestDar
```

Comment:
Same question here, can we leave the existing targets untouched?

Reply:
Not really, I think (see https://github.com/digital-asset/daml/pull/9143/files#diff-d5719626ac0f56dec36be3531a23a53ba46b74c7990ca1f862bab3e31255e6d0R689)